queue/queue.py

```diff
@@ -15,7 +15,16 @@ def enqueue(self, item):
 			self.size += 1
   
   def dequeue(self):
-    pass
+     if self.storage.head == None:
+			return None
+		else:
+			removed_node = Node(self.storage.head.get_value())
```

Comment:
Creating a node here is not necessary. You just need to grab the current head and return its value at the end.